release.py

```diff
@@ -421,16 +420,16 @@ def __init__(self, tag_name):
         self.level = data[3]
         self.serial = int(data[4])
         # This has the effect of normalizing the version.
-        self.text = "{}.{}.{}".format(self.major, self.minor, self.patch)
+        self.text = f"{self.major}.{self.minor}.{self.patch}"
```

Comment:
```suggestion
        self.text = self.normalized()
```
This might be out of the scope of this PR, since it's not linting-related but about removing duplication, so feel free to ignore or fix in a separate PR.  The comment above might also need to be updated.

Reply:
Yep, I'll add it to https://github.com/python/release-tools/pull/110 because it's also adding tests here 👍